### PR TITLE
added new command support in EclecticIQ pack

### PR DESCRIPTION
**Pull request by deepanshu-eiq**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added two new command in EclecticIQ pack
1. `eclecticiq-request-put` to make HTTP PUT request with EclecticIQ
2. `eclecticiq-request-patch` to make HTTP PATCH request with EclecticIQ

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by aaron1535** [comment]
Hi @deepanshu-eiq, 
Thanks for contribution
Please send me a link to a demo video that shows running the new commands you added.

**Comment by deepanshu-eiq** [comment]
[Uploading new_commands_demo.webm…]()

File: Packs/EclecticIQ/Integrations/EclecticIQIntelligenceCenterv3/EclecticIQIntelligenceCenterv3.py [hunk 1]
```diff
@@ -292,6 +292,16 @@ def send_api_request(self, method, path, params=None, data=None):
...
+            elif method == "patch":
+                r = requests.patch(
+                    url,
+                    headers=self.headers,
+                    params=params,
+                    data=json.dumps(data),
+                    verify=self.verify_ssl,
+                    proxies=self.proxies,
+                    timeout=30,
+                )
```
Comment (aaron1535) [inline Packs/EclecticIQ/Integrations/EclecticIQIntelligenceCenterv3/EclecticIQIntelligenceCenterv3.py:304]: The code is very long and really double
It seems advisable to make it more generic
something like that

```
def send_api_request(self, method, path, params=None, data=None):
    url = self.baseurl + path

    r = None
    try:
        if hasattr(requests, method):
            request_method = getattr(requests, method)
            r = request_method(
                url,
                headers=self.headers,
                params=params,
                data=json.dumps(data),
                verify=self.verify_ssl,
                proxies=self.proxies,
                timeout=30,
            )
        else:
            self.eiq_logging.error("Unknown method: " + str(method))
            raise Exception("Unsupported HTTP method: {}".format(method))
     

Continuation of the function
...

**Review by aaron1535 — CHANGES_REQUESTED** [review]
Hi @deepanshu-eiq,
Thank you for your contribution
Good job!
I have one suggestion to improve readability. If you want you can apply it.
You can mark it as "resolved" even without the change

File: Packs/EclecticIQ/Integrations/EclecticIQIntelligenceCenterv3/EclecticIQIntelligenceCenterv3.py (hunk 1 above)
Comment (deepanshu-eiq) [reply Packs/EclecticIQ/Integrations/EclecticIQIntelligenceCenterv3/EclecticIQIntelligenceCenterv3.py]: @aaron1535 it looks a nice change, thanks for raising it, I have updated the MR with this fix

**Comment by aaron1535** [comment]
@deepanshu-eiq Please update your Docker image to the latest version `3.11.10.115186`

**Comment by deepanshu-eiq** [comment]
@aaron1535 updated

**Review by aaron1535 — APPROVED** [review]

**Pull request merged**